### ISSUE TITLE
MB/Phasor: fix for reading AY regs (via 6522 PortA) after AY set to INACTIVE function

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#1193]
For mb-audit v0.1, the Apple II code was erroneously doing AY READs by reading 6522 PortA **after** setting the AY to INACTIVE.

This worked for a Mockingboard C, but not for a Phasor (in Phasor-native mode, ie. with all 4x AYs available).

From GI's AY-3-8910 data sheet, INACTIVE is defined as:
> The PSG/CPU bus is inactive. DA7-DA0 are in a high impedance state.

In mb-audit v0.1:
- For MB, the AY READs return the value written to the AY reg.
- For Phasor, the AY READs return 0xFF, which is what the mb-audit v0.1 code tests for.

NB. Both the MB & Phasor AY READ routines were fixed at mb-audit v0.2 to read PortA before setting the AY to INACTIVE.

All the mb-audit releases were tested & work on real h/w (MB-C and Phasor) so emulation should support them all too.

**Comment by tomcw** [comment]
Test using mb-audit v0.1 with a MB in slot 4 and a Phasor in slot 5.

**Comment by tomcw** [comment]
mb-audit v1.3 test for this read Port-A after INACTIVE: [374cb178b84e8b3f4823d822050bc7ce6e7531e5](https://github.com/tomcw/mb-audit/commit/374cb178b84e8b3f4823d822050bc7ce6e7531e5)

**Comment by tomcw** [comment]
>This worked for a Mockingboard C, but not for a Phasor (in Phasor-native mode, ie. with all 4x AYs available).

NB, the reason it worked for *my* Mockingboard C, is because I built it with WDC 6522's, ie. W65C22S6TPG-14.
This doesn't pull the bus high when the AY's aren't driving it.

I've changed the mb-audit test at v1.50 to pass for R6522 and warn for the WDC 6522 behaviour:
https://github.com/tomcw/mb-audit/commit/c02d29b4bdebd95c4dc556b81b68062511eacacf
